client/webserver/api.go

```diff
@@ -427,19 +407,26 @@ func (s *WebServer) apiOrder(w http.ResponseWriter, r *http.Request) {
 // apiReconfig sets new configuration details for the wallet.
 func (s *WebServer) apiReconfig(w http.ResponseWriter, r *http.Request) {
 	form := &struct {
-		AssetID uint32            `json:"assetID"`
-		Config  map[string]string `json:"config"`
-		AppPW   encode.PassBytes  `json:"pw"`
+		AssetID  uint32            `json:"assetID"`
+		Config   map[string]string `json:"config"`
+		SetNewPW bool              `json:"setNewPW"`
```

Comment:
Can eliminate this `SetNewPW` field using the `NewPW != nil` check in `ReconfigureWallet`